### PR TITLE
dev-resources

### PR DESCRIPTION
**Pull request by tijno**

### CODE REVIEW

**Comment by maebeam** [comment]
This is incredible! Thank you tijn!

**Review by maebeam — APPROVED** [review]

File: devs/resources.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* [Cloudflare Workers DESO Embed Code](https://github.com/DeSoDev/EmbeDeSo) - Enables DESO urls from showing good embed info on twitter, discord, and in google search.
+
+## Code examples
+
+* Use Identity login:
+
+    - [Javascript](https://github.com/mubashariqbal/login-with-bitclout)
+    - [React](https://github.com/BogdanDidenko/react-bitclout-login) 
+    - [Svelte](https://github.com/mvanhalen/bitclout-identity-window-svelte) 
+    - [Deno](https://github.com/PrismWeb3/deso-web-identity)
```
Comment (HPaulson) [inline devs/resources.md:63]:
```suggestion
    - [Javascript](https://github.com/PrismWeb3/deso-web-identity) 
```

Our package actually isn't written in or for Deno; It's build purely for web applications. It does utilizes some things from the  the Deso STD, but all of this is packaged together, so no need to actually use deno in any way.

The rest looks great -- thanks for this addition Tijn!

**Review by HPaulson — CHANGES_REQUESTED** [review]

**Comment by AeonSw4n** [comment]
Wow this is amazing! Thank you so much @tijno!

File: devs/resources.md (hunk 1 above)
Comment (tijno) [reply devs/resources.md:63]: got merged before i could change this so ill add this in update in a bit

**Comment by christophercelaya** [comment]
Woah, this is incredible! 
It really takes quite a bit of stress while searching the web for resources. 

Many thanks for taking the time to put this all together.

**Pull request merged**